app/lib/common/themes/colors/color_scheme.dart

```diff
@@ -43,8 +43,8 @@ var colorScheme = ColorScheme.dark(
   onPrimaryContainer: whiteColor,
   secondaryContainer: blackColor,
   onSecondaryContainer: whiteColor,
-  background: darkBlueColor,
-  onBackground: whiteColor,
-  surface: lightBlueColor,
-  onSurface: whiteBlueColor,
+  surface: darkBlueColor,
+  onSurface: whiteColor,
+  surfaceContainerHighest: lightBlueColor,
+  onSurfaceVariant: whiteBlueColor,
```

Comment:
I understood that by this change it will not impact changes related to background and surface but is this mandatory in new Flutter 3.22?

Reply:
Yes, as per my above comment! ;)